api/src/test/resources/io/strimzi/api/kafka/model/Kafka-with-entity-operator.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: strimzi-ephemeral
+spec:
+  kafka:
+    replicas: 3
+    storage:
+      type: persistent-claim
+      size: 500Gi
+    listeners:
+      plain: {}
+      tls: {}
+  zookeeper:
+    replicas: 3
+    storage:
+      type: ephemeral
+  entityOperator:
+    topicOperator:
+      watchedNamespaces: my-ns
+      reconciliationIntervalSeconds: 60
+    userOperator:
+      watchedNamespaces: my-other-ns
+      reconciliationIntervalMilliseconds: 20000
```

Comment:
reconciliationIntervalMilliseconds -> reconciliationIntervalSeconds?

Reply:
why the test didn't fail. A not existing field in the CRD is just ignored? Btw I have updated it.